partner_other_names/models/res_partner.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+# Copyright 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Guillermo Montoya <Github@guillermm>
+# Copyright 2018 Joan Marín <Github@JoanMarin>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.addons.partner_firstname import exceptions
+
+
+class ResPartner(models.Model):
+    """Adds other names."""
+    _inherit = "res.partner"
+
+    othernames = fields.Char("Other Names")
+
+    @api.model
+    def _get_computed_name(self, firstname, othernames, lastname, lastname2):
```

Comment:
You could add default argument values here to increase resilience.
```suggestion
    def _get_computed_name(self, firstname=None, othernames=None, lastname=None, lastname2=None):
```

Reply:
You didn't change this...

Reply:
You're right. We already did.